src/testdir/test_environ.vim

```diff
@@ -0,0 +1,44 @@
+scriptencoding utf-8
+
+function Test_environ()
```

Comment:
Most other tests uses `func` instead of `function`.
(Same for other functions.)